docs/diagrams/UiClassDiagram.puml

```diff
@@ -13,6 +13,8 @@ Class HelpWindow
 Class ResultDisplay
 Class PersonListPanel
 Class PersonCard
+Class HistoryListPanel
```

Comment:
Should we export these changes to a png and replace the current one?

Reply:
ya, my bad